index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Fahim's Portfolio</title>
+  </head>
+  <body>
+    <div class="pf-wrap">
+      <div class="pf-header">
+        <a class="pf-logo" href=""><img class="pf-logo" src="../portfolio-website/img/F_logo.png" alt="Fahim logo" /></a>
+        <span class="material-symbols-outlined"> menu </span>
+        <!-- <a class="mb-menu-button" href="#"><img src="../portfolio-website/img/mobile-menu.svg" alt="mobile-menu" /></a> -->
+      </div>
+      <div class="pf-headline">
+        <div class="contents">
+          <h2>I’m Mariane Glad to see you!</h2>
+          <p>
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <div class="social-link">
+            <p>LET’S CONNECT</p>
+            <ul>
+              <li>
+                <a href="#"><img src="../portfolio-website/img/twitter.png" alt="Twitter Icon" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="../portfolio-website/img/linkedin.png" alt="Linkedin Icon" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="../portfolio-website/img/m-icon.png" alt="M Icon" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="../portfolio-website/img/git.png" alt="Github Icon" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="../portfolio-website/img/v-icon.png" alt="V Icon" /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
```

Comment:
- I'd like to kindly suggest that the container of this section  highlighted is a `section` tag because It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.

- Kindly ensure that the `<p>LET’S CONNECT</p>` has an anchor tag or a button because this is meant to be a clickable link navigating the user to a different page.

- It is important for `<h2>I’m Mariane Glad to see you!</h2>` to be replaced with `<h1>I’m Mariane Glad to see you!</h1>` because it it your portfolio's main title and it is okay to replace the name `Mariane` with yours.
Also, let's ensure to stick with the font color on Figma 🌸 

- Your logo and social icon images are currently not being referenced properly, hence causing them to be broken on the browser. Can we fix this? You can simply remove the `../portfolio-website` and leave just this 👉🏽 `<img src="./img/twitter.png" alt="Twitter Icon" />`. 


Here is what I mean 👇🏽 

| Your Design  |Figma Design |
| :---:   | :---: | 
| <img width="388" alt="Screenshot 2023-01-12 at 07 46 42" src="https://user-images.githubusercontent.com/99246359/212083361-b85ac688-8dbc-4162-9591-cdd2222a5974.png"> |<img width="374" alt="Screenshot 2023-01-12 at 00 19 39" src="https://user-images.githubusercontent.com/99246359/212083468-e32319e9-22dd-467d-b755-c4d9dc42f3fa.png">  |